xml/ay_custom_user_scripts.xml

```diff
@@ -689,9 +695,14 @@ echo Hi
      and the output logs in the <filename>log</filename> directory.
     </para>
     <para>
-     The log consists of the output produced when executing the shell
-     scripts using the following command:
+     The log consists of the output produced when executing the
+     scripts containing both standart output and standard error output
+     together.
```

Comment:
```suggestion
     The log consists of the output produced when executing the
     scripts, containing both the standard output and the standard
     error output combined.
```

Reply:
sounds good to me. Feel free to improve phrasing as much as you want :wink: